### ISSUE TITLE
How to migrate data between Gitea servers?

### ISSUE THREAD

**Issue by springeye** [go-gitea/gitea#30507]
### Feature Description

My old server is Centosos, and my new server is Docker Gitea,
I have tried many methods, but it took me a few days without success. Either the Docker container cannot start after the migration is completed, or it cannot be accessed after successful startup without any errors
I also checked a lot of fragmented information, such as [https://github.com/go-gitea/gitea/issues/7858](https://github.com/go-gitea/gitea/issues/7858), and it seems that no one is useful either

Can  provide a command to migrate?? such as gitea -- migrate newsever.com newadmintoken

**Comment by 99rgosse** [comment]
Have you tried using the gitea backup and restore command ? 
https://docs.gitea.com/administration/backup-and-restore#using-docker-dump

In docker, you just have to copy the data from the docker volume to your new system and put it in the same docker volume

It works flawlessly everyday for me, I recreate the running full server on a spare one.

**Comment by springeye** [comment]
> Have you tried using the gitea backup and restore command ? https://docs.gitea.com/administration/backup-and-restore#using-docker-dump
> 
> In docker, you just have to copy the data from the docker volume to your new system and put it in the same docker volume
> 
> It works flawlessly everyday for me, I recreate the running full server on a spare one.

Yes, I have tried before, but the first few attempts have all failed (without any error messages or normal access). Yesterday, I manually processed the files and permissions and finally managed to use them

**Comment by springeye** [comment]
I hope Gitea can backup and restore with just one click. Whether it's native app or docker
